### PR TITLE
RAVTDBQ-171 Support for Default Constraint

### PR DESCRIPTION
**Pull request by Mahesh-Raut**
Added support to create SqlNode for TIMESTAMP WITH TIME ZONE datatype.

### CODE REVIEW

File: core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    TimestampString timestampString = new TimestampString("2020-05-21 11:20:01.4321");
+
+    final SqlNode node = SqlLiteral.createTimestamp(timestampString, 4,
+        SqlParserPos.ZERO);
+    final String expectedSqlNode = "TIMESTAMP '2020-05-21 11:20:01.4321'";
+
+    assertEquals(node.toString(), expectedSqlNode);
+  }
+
+  @Test void testTimestampWithTimeZoneLiteralSqlNode() {
```
Comment (akash-pawar1) [inline core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java:44]: Please create a generic sqlNode for timestamp with time zone .

Current :- "2020-05-21 11:20:01 GMT-05:00";
Expected  :-  timestamp '2020-05-21 11:20:01-05:00';

File: core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    String abc = "2020-05-21 11:20:01 GMT-05:00";
+    TimestampWithTimeZoneString timestampWithTimeZoneString = new TimestampWithTimeZoneString(abc);
+
+    final SqlNode node = SqlLiteral.createTimestampWithTimeZone(timestampWithTimeZoneString, 4,
+        SqlParserPos.ZERO);
```
Comment (akash-pawar1) [inline core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java:49]: If possible try to remove the hardcoded precision value.

Comment (akash-pawar1) [reply core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java:49]: you can refer

SqlParserUtil.parseTimeLiteral

File: core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java (hunk 1 above)
Comment (Mahesh-Raut) [reply core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java:48]: Changes done

File: core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java (hunk 2 above)
Comment (Mahesh-Raut) [reply core/src/test/java/org/apache/calcite/unparse/TimestampLiteralSqlNodeTest.java:53]: We currently have not handled it.
For this, we create the new Jira.

**Comment by akash-pawar1** [comment]
L1 done. The changes look good to me.

**Review by deepakkadam-dm — APPROVED** [review]
Looks Ok

**Pull request merged**